querydsl-jpa/pom.xml

```diff
@@ -48,30 +48,13 @@
         </exclusion>
       </exclusions>      
       <optional>true</optional>
-    </dependency>
-    <dependency>
-      <groupId>org.hibernate</groupId>
-      <artifactId>hibernate-entitymanager</artifactId>
-      <version>${hibernate.version}</version>
-      <optional>true</optional>
+      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>org.hibernate.validator</groupId>
       <artifactId>hibernate-validator</artifactId>
       <version>${hibernate.validator.version}</version>
-      <optional>true</optional>
-    </dependency>
-    <dependency>
-      <groupId>javax.el</groupId>
-      <artifactId>javax.el-api</artifactId>
```

Comment:
Remove unused dependency to el-api API (it still comes transitively in the test scope through `hibernate-validator`).